src/webextension/background/datastore.js

```diff
@@ -20,9 +20,11 @@ async function recordMetric(method, itemid, fields) {
   telemetry.recordEvent(method, "datastore", extra);
 }
 
-export default async function openDataStore() {
+export default async function openDataStore(cfg) {
   if (!datastore) {
+    cfg = cfg || {};
```

Comment:
Instead of this, I think we could just define the function as `openDataStore(cfg = {})`.